### DISCUSSION TITLE
<Utility.utils.obj object at ...>  error

### DISCUSSION THREAD

**Discussion by sc303030** [pydantic/pydantic#9033] [Question] [answered]
There is no error in fastapi 0.99 version, but in version 0.110.0, there is an error: Unable to serialize unkown type: <class 'Utility.utils.obj'> I think it's probably because the version has been updated, how do I fix it??
``` shell
pydantic_core._pydantic_core.PydanticSerializationError: Unable to serialize unknown type: <class 'Utility.utils.obj'>
```

**Comment by uriyyo** [comment, marked as answer]
Is `'Utility.utils.obj` your custom class? Then you will gonna need to add a serializer to it.

You can checkout docs, or provide us with more context)

https://docs.pydantic.dev/latest/concepts/types/#custom-types
